src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$circle-size: 75vh;
+$center-size: 5%;
+
+$c-bg-color: #808080;
+$c-seconds-hand: #0ff;
+$c-minutes-hand: #ff0;
+$c-white: #fff;
```

Comment:
Be consistent with your color codes. 
All colors using 6-digit hex code or all colors using 3-digit hex code